examples/get_alerts.rb

```diff
@@ -0,0 +1,10 @@
+require 'prometheus/alert_client'
+
+# returns a client
+prometheus = Prometheus::AlertClient.client(url: 'http://localhost:9099', path: '/topics/zgalor')
```

Comment:
I prefer the kubeclient style, I think it is more idiomatic:
```
client = Kubeclient::Client.new(...)
```

Reply:
It's the same. You can access it the same way:
`Prometheus::AlertClient::Client.new(...)`

I could cancel the layering and have `PrometheusAlertClient::Client.new(...)`
but this way it's more consistent with `Prometheus::ApiClient`

Reply:
@zgalor I suggested 
```
Prometheus::AlertClient.new(..)
```

Up to you...